tests/rptest/tests/pandaproxy_test.py

```diff
@@ -208,7 +211,6 @@ def __init__(self, context, **kwargs):
         super(PandaProxyEndpoints, self).__init__(
             context,
             num_brokers=3,
-            enable_pp=True,
```

Comment:
nit: It feels like a default constructed `PandaproxyConfig` is simpler here than in `PandaProxyTestMethods` and `PandaProxySASLTest`